eth_keys/tools/factories.py

```diff
@@ -0,0 +1,33 @@
+try:
+    import factory
+except ImportError as err:
+    raise ImportError(
+        "Use of `eth_keys.tools.factories` requires the `factory-boy` package "
```

Comment:
It's probable that I'm misunderstanding here, but why wouldn't we just add `factory-boy` to the required libraries in `setup.py`?

Reply:
Because it isn't necessary for normal operation of the library.  These factories are strictly intended for testing purposes.

This is roughly the philosophy behind the `<library>.tools` namespace across our various codebases.  It is the place to put stuff that we **do** want to be type checked, linted, etc, and that *may* be useful to others but that **is NOT** in any way critical to the library code.